Cargo.toml

```diff
@@ -22,15 +22,15 @@ ash = { version = "0.29", optional = true }
 simdeez = { version = "0.6.1", optional = true }
 ncollide3d = { version = "0.21", optional = true }
 nphysics3d = { version = "0.13.0", optional = true }
-lru = { version = "0.1", optional = true }
+lru = { version = "0.4", optional = true }
 
 [dev-dependencies]
 winit = "0.18.1"
 criterion = "0.2.10"
 vk-shader-macros = "0.2"
 memoffset = "0.2.1"
-half = "1.3.0"
-noise = "0.5.1"
+half = "1.4.0"
+noise = { version = "0.6.0", features = [] }
```

Comment:
Was this intended to be `default-features = false`?

Reply:
Yes, this is an error.

Reply:
fixed in commit
https://github.com/Ralith/planetmap/pull/2/commits/a6b58c7abb7805f4bbac7e58279bb894074248e9